adafruit_sdcard.py

```diff
@@ -253,6 +254,7 @@ def _cmd(self, cmd, arg=0, crc=0, response_buf=None, data_block=True, wait=True)
                     if response_buf:
                         if data_block:
                             # Wait for the start block byte
+                            spi.readinto(buf, start=1, end=2, write_value=0xff)
```

Comment:
Does this really speed things up? The loop test should be pretty quick. Duplicating the line makes maintenance a bit more difficult.

Reply:
The speed gain is mesurable, but very small. And may be dependent on a specific card timing, I havent't made systematics tests. I agree that it may not worth the hassle and will revert those changes, with just a proper initialization of buf[1].

Reply:
Didn't change https://github.com/adafruit/Adafruit_CircuitPython_SD/blob/c7aa2f225ba002d55f38d558e33b68faf8adde5a/adafruit_sdcard.py#L388 
because it was like this in the original file.